src/modules/CurrencyModule.cpp

```diff
@@ -902,7 +902,219 @@ CurrencyModule::CurrencyModule(UmikoBot* client) : Module("currency", true), m_c
+							return;
+						}
+
+						if (args.size() == 1 || args.size() > 2)
+						{
+							client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+							return;
+						}
+
+						if (args.size() == 2)
+						{
+							auto& config = getServerData(channel.guildId());
+							config.bribeSuccessChance = args.at(1).toInt();
+							client.createMessage(message.channelId(), "Bribe Success Chance set to **" + QString::number(config.bribeSuccessChance) + "%**");
+						}
+
+					});
+			});
+		RegisterCommand(Commands::CURRENCY_SET_MAX_BRIBE_AMOUNT, "setmaxbribeamount", [this](Discord::Client& client, const Discord::Message& message, const Discord::Channel& channel)
+			{
+
+				QStringList args = message.content().split(' ');
+
+				Permissions::ContainsPermission(client, channel.guildId(), message.author().id(), CommandPermission::ADMIN,
+					[this, args, &client, message, channel](bool result)
+					{
+						GuildSetting* setting = &GuildSettings::GetGuildSetting(channel.guildId());
+						if (!result)
+						{
+							client.createMessage(message.channelId(), "**You don't have permissions to use this command.**");
+							return;
+						}
+
+						if (args.size() == 1 || args.size() > 2)
+						{
+							client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+							return;
+						}
+
+						if (args.size() == 2)
+						{
+							auto& config = getServerData(channel.guildId());
+							config.bribeMaxAmount = args.at(1).toInt();
+							client.createMessage(message.channelId(), "Bribe Max Amount set to **" + QString::number(config.bribeMaxAmount) + "**");
+						}
+
+					});
+			});
+		RegisterCommand(Commands::CURRENCY_SET_LEAST_BRIBE_AMOUNT, "setleastbribeamount", [this](Discord::Client& client, const Discord::Message& message, const Discord::Channel& channel)
+			{
+
+				QStringList args = message.content().split(' ');
+
+				Permissions::ContainsPermission(client, channel.guildId(), message.author().id(), CommandPermission::ADMIN,
+					[this, args, &client, message, channel](bool result)
+					{
+						GuildSetting* setting = &GuildSettings::GetGuildSetting(channel.guildId());
+						if (!result)
+						{
+							client.createMessage(message.channelId(), "**You don't have permissions to use this command.**");
+							return;
+						}
+
+						if (args.size() == 1 || args.size() > 2)
+						{
+							client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+							return;
+						}
+
+						if (args.size() == 2)
+						{
+							auto& config = getServerData(channel.guildId());
+							config.bribeLeastAmount = args.at(1).toInt();
+							client.createMessage(message.channelId(), "Bribe Least Amount set to **" + QString::number(config.bribeLeastAmount) + "**");
+						}
+
+					});
+			});
+	//End*****************************************************************************************************************************************
```

Comment:
idt we need the start and end comments?

Reply:
That's a bribe code indicator, you can easily see that the bribe code stars where and ends where 😄

Reply:
yeah but it's ez to tell imo

Reply:
I agree with beastboioioi, we don't need the start and end comments

Reply:
Will I remove it?, if yes how?

Reply:
Okay I removed! Thanks for the suggestions.